### ISSUE TITLE
Problem With Executable Jar And plugin.yml

### ISSUE THREAD

**Issue by jeffbrown** [grails/grails-core#612]
It looks like the stand alone executable jars generated by Grails 3.0.1 aren't recognizing the plugin.yml config files contributed by plugins.  This may be what is causing grails3-plugins/quartz#4.

**Comment by zyro23** [comment]
this might be related: https://github.com/grails/grails-core/issues/607

**Comment by graemerocher** [comment]
Is this still valid?

**Comment by jwagenleitner** [comment]
I think this can be closed.  For Grails versions 3.0.0 and 3.0.1 I get the exception referenced in the quartz issue.  But Grails versions 3.0.2 and 3.0.8 start up fine with no exception and the `quartz.autoStartup` config property was displayed on the index page.

I performed the following to test with each version mentioned:

1.  Ran a `create-app`
1.  added `${grailsApplication.config.quartz.autoStartup}` just below body tag in `grails-app/views/index.gsp`
1.  added `compile 'org.grails.plugins:quartz:2.0.0.M4'` to dependencies in `build.gradle`
1.  Ran `./gradlew clean build`
1.  Ran `java -jar build/libs/app-0.1.jar`
